code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -56,10 +56,12 @@
 	if(istype(target, /obj/machinery/door/airlock))
 		if(!removingairlock)
 			src << "<span class='notice'>You start tearing apart the airlock...</span>"
+			playsound(src.loc, 'sound/machines/airlock_alien_prying.ogg', 100, 1)
+			sleep(20)
```

Comment:
I think now you can open multiple airlocks again because you dont set the opening var until after the sleep